server/cmd/mmctl/commands/utils/main.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+)
+
+func GetConfig(c client.Client, context context.Context) (*model.Config, error) {
+	config, response, err := c.GetConfig(context)
+	if err != nil || response.StatusCode < 200 || response.StatusCode >= 300 {
+		if err != nil {
+			return nil, err
+		}
+		return nil, errors.New(fmt.Sprintf("Wrong response client: code: %d", response.StatusCode))
+	}
```

Comment:
`GetConfig` already checks for errors (https://github.com/mattermost/mattermost/blob/79e1c182ce435e8b946535a4344de260a31f504e/server/public/model/client4.go#L763-L798)

Do we need the custom code?

Reply:
So, we're sure.

Reply:
I think we can completely remove the `utils.GetConfig` function. `client4.GetConfig` does everything we need.

Reply:
I did so